app/src/main/java/org/sopt/android_hyorim_30th/ui/home/HomeViewModel.kt

```diff
@@ -0,0 +1,19 @@
+package org.sopt.android_hyorim_30th.ui.home
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import org.sopt.android_hyorim_30th.R
+
+class HomeViewModel : ViewModel() {
+    private val _profileImage = MutableLiveData(R.drawable.loopy)
+    val profileImage: LiveData<Int> = _profileImage
```

Comment:
```suggestion
    val profileImage: LiveData<Int> get() = _profileImage
```
저는 보통 거의 대부분 `get()` 을 써서 가져오는데, 혹시 `=` 로 바로 할당해준 이유가 있나요 ?

Reply:
`get()` 을 쓰면 `profileImage`를 참조할 때 `_profileImage` 를 반환해주는 함수가 계속 호출되는 것으로 큰 의미가 없고 그냥 참조를 하고 있어도 됩니다.

Reply:
@mym0404 그러면 여기서는 val 변수들이라서 값이 바뀔 이유가 없으니까 `get()` 을 굳이 쓰지 않아도 된다 이런 맥락으로 이해하면 될까요?